tests/Model/FormModelTest.php

```diff
@@ -95,7 +96,9 @@ public function throwingTypeErrorWhenPassedNonStringAction()
      */
     public function getFieldsReturnsProperListOfFields()
     {
-        $fields = [$this->field, $this->field];
+        $field2 = clone $this->field;
+        $field2->setName("two");
+        $fields = ["one" => $this->field, "two" => $field2];
```

Comment:
why are you modifying existing test? it is antipattern.

Reply:
Test was badly designed. we cannot have two fields with same name.

Reply:
okay.